docs/blog/posts/2023-12-06-feature_update.md

```diff
@@ -0,0 +1,119 @@
+
+Note, labelled data is required to generate this chart.
+
+### :material-table: Completeness Chart
+
+When linking multiple datasets together, one of the most important factors for a successful linkage is the number of common fields across the datasets. 
+
+Splink now has the [completeness chart](../../charts/completeness_chart.ipynb) which gives a simple view of how well populated fields are across datasets.
+
+[![](./img/completeness_chart.png)](../../charts/completeness_chart.ipynb)
+
+
+## :clipboard: Settings Validation
+
+The [Settings dictionary](../../settings_dict_guide.md) is central to everything in Splink. It defines everything from the sql dialect of your backend to how features are compared in Splink model. 
+
+A common sticking point with users is how easy it is to make small errors when defining the Settings dictionary, resulting in unhelpful error messages.
+
+To address this issue, the [Settings Validator](../../dev_guides/settings_validation/settings_validation_overview.md) provides clear, user-friendly feedback on what the issue is and how to fix it.
+
+
+## :simple-adblock: Blocking Rule Library (Improved)
+
+In our [previous blog](../posts/2023-12-06-feature_update.md#no_entry_sign-drop-support-for-python-37) we introduced the Blocking Rule Library (BRL) built upon the `exact_match_rule` function. When testing this functionality we found it pretty verbose, particularly when blocking on multiple columns, so figured we could do better. From Splink v3.9.6 we introduced the `block_on` function to supercede `exact_match_rule`.
+
+For example, a block on `first_name` and `surname` now looks like:
+
+```py
+from splink.duckdb.blocking_rule_library import block_on
+block_on(["first_name", "surname"])  
+```
+
+as opposed to
+
+```py
+import splink.duckdb.blocking_rule_library as brl
+brl.and_(
+  brl.exact_match_rule("first_name"),
+  brl.exact_match_rule("surname")
+)
+```
+
+All of the [tutorials](../../demos/tutorials/03_Blocking.ipynb), [example notebooks](../../demos/examples/examples_index.md) and [API docs](../../blocking_rule_library.md) have been updated to use `block_on`.
+
+## :electric_plug: Backend Specific Installs
+
+Some users have had difficulties downloading Splink due to additional dependencies, some of which may not be relevant for the backend they are using. To solve this, you can now install a minimal version of Splink for your given SQL engine.
+
+For example, to install Splink purely for Spark use the command:
+
+```bsh
+pip install 'splink[spark]'
+```
+
+See the [Getting Started page](../../getting_started.md#backend-specific-installs) for further guidance.
+
+## :no_entry_sign: Drop support for python 3.7
+
+From Splink 3.9.7, support has been dropped for python 3.7. This decision has been made to manage dependency clashes in the back end of Splink.
+
+If you are working with python 3.7, please revert to Splink 3.9.6.
+
+```bsh
+pip install splink==3.9.6
+```
```

Comment:
I know I mentioned that we support some version of python 3.7 (>=3.7.1), but I think we'll have to upgrade to 3.8 soon anyway, so this section looks good.